cloudify_cli/env.py

```diff
@@ -548,22 +548,34 @@ def do_request(self, *args, **kwargs):
         if kwargs.get('timeout') is None:
             kwargs['timeout'] = self.default_timeout_sec
 
+        # First try with the main manager ip given when creating the profile
+        # with `cfy profiles use`
+        self.host = self._profile.manager_ip
+        response = self._try_do_request(*args, **kwargs)
+        if response:
+            return response
+
         for node_index, node in list(enumerate(
```

Comment:
This will repeat the first server twice, no?
And how did this behavior changed?